config/dataset/nerf_synthetic.yaml

```diff
@@ -1,3 +1,3 @@
 _target_: neddf.dataset.NeRFSyntheticDataset
-dataset_dir: "data/nerf_synthetic/chair/"
+dataset_dir: "data/bunny_smoke/"
```

Comment:
Although still using `NeRFSyntheticDataset` however this is `bunny_smoke` dataset, right? So I feel it might be better to change filename from `nerf_synthetic` to `bunny_smoke` or something.

Reply:
I assumed NeRFSyntheticDataset as the data format...
I am sure it is confusing, so would it be natural to add a new configuration file for bunny_smoke?

Reply:
I think using class `NeRFSyntheticDataset` as data format of bunny_smoke dataset is OK. But config name should be different. If I were user, I expect `poetry run python3 neddf/scripts/run.py dataset=nerf_synthetic` kicks training job agaist NeRF synthetic Dataset (not bunny smoke dataset).

Reply:
>  would it be natural to add a new configuration file for bunny_smoke?
Yeah, I think so.